src/Form/molecules/TextareaField.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { TextareaHTMLAttributes } from 'react';
+import styled from 'styled-components';
```

Comment:
This import isn't needed as it's not used.

Reply:
Hi! In this case it's needed becase I'm sending props down to TextArea.
The specific scenario of sending "placeholder" as prop shows a error message.